### PR TITLE
pkg/umorse: Add kconfig option

### PR DESCRIPTION
**Pull request by MrKevinWeiss**
### Contribution description

Add the umorse pkg to the pkg rsource and expose dependencies.

### Testing procedure

`TEST_KCONFIG=1 make menuconfig -C tests/pkg_umorse/`

### Issues/PRs references

Split from #15950

### CODE REVIEW

File: pkg/umorse/Kconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+config USEPKG_UMORSE
+    bool "umorse"
+    select USEMODULE_POSIX_SLEEP
```
Comment (leandrolanzieri) [inline pkg/umorse/Kconfig:9]:
```suggestion
    select MODULE_POSIX_SLEEP
```

File: pkg/umorse/Kconfig [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+config USEPKG_UMORSE
```
Comment (leandrolanzieri) [inline pkg/umorse/Kconfig:7]:
```suggestion
config PACKAGE_UMORSE
```

File: pkg/umorse/Kconfig (hunk 1 above)
Comment (leandrolanzieri) [reply pkg/umorse/Kconfig:9]: We should check if it is safe to keep the `select` or this should be `depends on`. The posix modules are not modelled AFAIK, so those should be added.

File: pkg/umorse/Kconfig [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    bool "umorse"
+    select USEMODULE_POSIX_SLEEP
+
```
Comment (leandrolanzieri) [inline pkg/umorse/Kconfig:10]: Check this new blank line error please.

**Review by leandrolanzieri — COMMENTED** [review]
You should include the `pkg_umorse` test application to the ones that murdock tests with Kconfig
https://github.com/RIOT-OS/RIOT/blob/d7f3e92c308b7922d8f3f5170e14cdefa60b1769/.murdock#L27-L28

With the corresponding `app.config.test`.

**Comment by MrKevinWeiss** [comment]
Done and squashed!

**Comment by MrKevinWeiss** [comment]
Don't know why I have 2 of these #15985

**Comment by miri64** [comment]
I was confused when I saw this in my MUA and only then I understood.

**Pull request closed without merging**